cmd/krel/cmd/automate_aftercode.go

```diff
@@ -0,0 +1,77 @@
+/*
+Copyright 2020 The Kubernetes Authors.
```

Comment:
```suggestion
Copyright 2021 The Kubernetes Authors.
```
(same for other occurrences)